cypress/support/fragments/circulation/newPatronNoticePolicies.js

```diff
@@ -0,0 +1,54 @@
+import getRandomPostfix from '../../utils/stringTools';
+import { Button, TextField, TextArea, NavListItem, Checkbox } from '../../../../interactors';
+// TO DO: update test with duplicate and edit methods, after PO will review test case.
+const actionsButton = Button('Actions');
+const nameField = TextField({ id: 'notice_policy_name' });
+
+export default {
+  defaultUiPatronNoticePolicies : {
+    name: `Test_notice_${getRandomPostfix()}`,
+    description: 'Created by autotest team',
+  },
+  createPolicy(patronNoticePolicy) {
+    cy.do(Button({ id: 'clickable-create-entry' }).click());
+    this.fillGeneralInformation(patronNoticePolicy);
+  },
+  fillGeneralInformation: (patronNoticePolicy) => {
+    cy.do([
+      nameField.fillIn(patronNoticePolicy.name),
+      Checkbox({ id: 'notice_policy_active' }).click(),
+      TextArea({ id: 'notice_policy_description' }).fillIn(patronNoticePolicy.description),
+      Button({ id: 'footer-save-entity' }).click(),
```

Comment:
method named as fill.... but it has saving in these two line. I suggest to move saving into separated methods